argocd/product-item-relationship-service-frontend/read-write/kustomization.yaml

```
@@ -9,3 +9,5 @@ resources:
 
 patchesStrategicMerge:
   - resources/argo-project.yaml
+  - argocd-notifications-cm.yaml
+  - argocd-notifications-secret.yaml
```

It's complaining for missing newline at the end of file, other than that, changes are OK.

```suggestion
  - argocd-notifications-secret.yaml

```